### PR TITLE
Preserve page size selection when changing filters

### LINKED ISSUE #36: Selected page size in the feedback list is forgotten when new searches are made
In the filter feedback page (feedback list):
If the teacher has selected a page size in the `paginate_by` select menu, then the selected page size is lost when the teacher makes a new query in the filter feedback form. The teacher needs to select the page size again each time he/she changes the filter parameters. That is frustrating for usability. When the filter feedback form is submitted, could the old `paginate_by` GET query parameter be kept in the URL so that the teacher's chosen page size is not repeatedly forgotten?

From the pull request comment:
https://github.com/apluslms/mooc-jutut/pull/26#pullrequestreview-1106176510

One approach could be to store the `paginate_by` value to the Django cache when the user changes it in the dropdown.
Another approach could be to add the page size parameter to the feedback filter form and its value in the form could be automatically updated from the `paginate_by` URL query parameter when the form is opened.

### PR DESCRIPTION
**Pull request by Mikael-Lenander**
# Description

**What?**

Page size is selected in the `paginate_by` select menu. Previously the page size was forgotten when the teacher made a new query in the filter feedback form. Now the page size is preserved.

**Why?**

Improves usability.

**How?**

Adds a new filter to the FeedbackFilter class. This filter does not actually filter anything. It merely stores the paginate_by parameter in the url. The form field is hidden, since we do not need a duplicate page size input field to the filter form.

Fixes #36

# Testing

**Remember to add or update unit tests for new features and changes.**

* How to [test your changes in A-plus](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations)
* How to [test accessibility](https://wiki.aalto.fi/display/EDIT/How+to+check+the+accessibility+of+pull+requests)

**What type of test did you run?**

- [ ] Accessibility test using the [WAVE](https://wave.webaim.org/extension/) extension.
- [ ] Django unit tests.
- [ ] Selenium tests.
- [ ] Other test. *(Add a description below)*
- [x] Manual testing.

Changed filters in the filter form and made sure that the page size setting is preserved. Changed the page size setting and made sure that the filters are preserved.

**Did you test the changes in**

- [x] Chrome
- [ ] Firefox
- [ ] This pull request cannot be tested in the browser.

# Translation

- [ ] Did you modify or add new strings in the user interface? ([Read about how to create translation](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations))

# Programming style

- [x] Did you follow our [style guides](https://apluslms.github.io/contribute/styleguides/)?
- [ ] Did you use Python type hinting in all functions that you added or edited? ([type hints](https://docs.python.org/3/library/typing.html) for function parameters and return values)

# Have you updated the README or other relevant documentation?

- [ ] documents inside the doc directory.
- [ ] README.md.
- [ ] Aplus Manual.
- [ ] Other documentation (mention below which documentation).

# Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

*Clean up your git commit history before submitting the pull request!*

### CODE REVIEW

File: feedback/filters.py [hunk 1]
```diff
@@ -272,7 +281,7 @@ class FeedbackFilter(django_filters.FilterSet):
     order_by = OrderingFilter(label=_("Sort"),
                               choices=ORDER_BY_CHOICE,
                               initial=ORDER_BY_DEFAULT)
-
+    paginate_by = PaginateByFilter(label=_("Paginate by"))
```
Comment (etanttila) [inline feedback/filters.py:284]: The string "Paginate by" doesn't exist yet in the localization file, so I would suggest updating that as well. You can find instructions for updating translations in the [a-plus repository documentation](https://github.com/apluslms/a-plus/tree/master/doc#updating-translations-for-finnish-and-english-versions) (although in MOOC-Jutut there is only the Finnish localization file, but the same command works).

**Review by etanttila — COMMENTED** [review]
These changes otherwise seem good to me(and work with a little testing), but the change adds a translatable string, which isn't translated in the localization file.

**Comment by etanttila** [comment]
This seems good to me and ready for merging.

**Comment by ihalaij1** [comment]
Looks good!

**Comment by etanttila** [comment]
After merging, it turned out that the code doesn't pass lint tests due to a renamed argument (see https://github.com/apluslms/mooc-jutut/actions/runs/9096725994/job/25002881344#step:5:26). It seems like the lint tests didn't run in your fork @Mikael-Lenander, which is why the problem wasn't noticed earlier. It would be valuable to get them working, so the tests could be run also for your other PRs to avoid the same problem in the future. Are you able to get them working on your own or do you need help with that?

**Comment by Mikael-Lenander** [comment]
I got them working. Thanks.

**Pull request merged**